source/extensions/filters/http/common/aws/signer_impl.cc

```diff
@@ -19,13 +19,22 @@ namespace Common {
 namespace Aws {
 
 void SignerImpl::sign(Http::Message& message, bool sign_body) {
+  const auto content_hash = createContentHash(message, sign_body);
```

Comment:
In a follow up can you move this code to source/extensions/common ?

Reply:
Sure

Reply:
Left a TODO, will follow-up with a PR shortly after this is merged.